pytorch_lightning/utilities/cli.py

```diff
@@ -165,7 +166,7 @@ def __init__(
         self.add_arguments_to_parser(self.parser)
         self.parse_arguments()
         if self.config['seed_everything'] is not None:
-            seed_everything(self.config['seed_everything'])
+            seed_everything(self.config['seed_everything'], workers=True)
```

Comment:
so CLI users get a different default here?

Reply:
It is the sane default.

And considering the CLI is supposed to handle it for you, I think it makes sense to have it set to True here.